src/lib.rs

```diff
@@ -232,9 +230,9 @@ pub enum ContentType {
 
 impl Copy for ContentType {}
 
-impl Into<i32> for ContentType {
-    fn into(self) -> i32 {
-        match self {
+impl From<ContentType> for i32 {
```

Comment:
As the documentation say, this is not the right way to impl a convert trait, implementing the `From` trait is better than the [`Into`](https://doc.rust-lang.org/std/convert/trait.Into.html) one. (if I understand you question correctly 😄)